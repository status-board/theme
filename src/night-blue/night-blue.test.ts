import nightBlue from './index';

test('Night Blue should match snapshot', (): void => {
    expect(nightBlue).toMatchSnapshot();
});
