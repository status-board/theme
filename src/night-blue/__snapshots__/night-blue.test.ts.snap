// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Night Blue should match snapshot 1`] = `
Object {
  "charts": Object {
    "axis": Object {
      "legendColor": "#eedba5",
      "textColor": "#eedba5",
      "tickColor": "#eedba5",
    },
    "colors": Array [
      "#FDECA4",
      "#cbb04a",
      "#D68649",
      "#648e9c",
      "#253445",
    ],
    "grid": Object {
      "stroke": "#25303d",
    },
    "tooltip": Object {
      "background": "#323f53",
      "borderRadius": "2px",
      "boxShadow": "0 1px 3px rgba(0, 0, 0, 0.5)",
      "color": "#f6ecd0",
      "fontSize": "1.4vmin",
    },
  },
  "colors": Object {
    "background": "#1e2430",
    "failure": "#de5029",
    "icon": "#e0c671",
    "success": "#4ec2b4",
    "text": "#eedba5",
    "textHighlight": "#f6ecd0",
    "unknown": "#495b71",
    "warning": "#d1be65",
  },
  "dashboard": Object {
    "header": Object {
      "background": "#2b3847",
      "boxShadow": "0 1px 1px rgba(0, 0, 0, 0.35)",
      "color": "#eedba5",
      "title": Object {},
    },
  },
  "label": Object {
    "addon": Object {
      "background": "#1e2836",
      "color": "#e0c671",
    },
    "extend": "
        border-radius: 2px;
        ",
    "main": Object {
      "background": "#212e41",
      "color": "#e0c671",
    },
  },
  "list": Object {
    "item": Object {
      "extend": "
            border-bottom: 1px solid #28323f;
            &:last-child {
                border-bottom: 0;
            }
            ",
      "hover": Object {
        "background": "#2f3d4b",
      },
      "meta": Object {
        "color": "#879db7",
      },
    },
  },
  "name": "night blue",
  "notifications": Object {
    "item": Object {
      "background": "#1e2430",
      "color": "#eedba5",
      "extend": "
            box-shadow: 0 1px 2px rgba(0, 0, 0, 0.15), 0 10px 20px rgba(0, 0, 0, 0.15);
            border-radius: 2px;
            ",
      "padding": "1.2vmin 2vmin",
    },
  },
  "root": Object {
    "background": "#1e2430",
    "color": "#eedba5",
    "extend": "
        @import url('https://fonts.googleapis.com/css?family=Raleway:200,400,600,800|Montserrat:400,700');
        
        & a {
            color: #f6ecd0;
        }
        ",
  },
  "typography": Object {
    "default": Object {
      "default": Object {
        "fontFamily": "'Raleway', sans-serif",
        "fontSize": "1.8vmin",
        "fontWeight": 400,
        "lineHeight": "1.5em",
      },
      "small": Object {
        "fontSize": "1.6vmin",
      },
      "strong": Object {
        "fontWeight": 600,
      },
    },
    "display": Object {
      "default": Object {
        "fontFamily": "'Montserrat', sans-serif",
        "fontSize": "1.8vmin",
        "lineHeight": "1.4em",
      },
    },
  },
  "widget": Object {
    "background": "#2b3847",
    "body": Object {
      "top": "5vmin",
    },
    "extend": "
        border-radius: 2px;
        box-shadow: 0 1px 1px rgba(0, 0, 0, .35);
        ",
    "header": Object {
      "background": "#323f53",
      "color": "#eedba5",
      "count": Object {
        "color": "#7e9ebc",
        "extend": "
                background-color: #1e2836;
                box-shadow: 0 1px 0 rgba(0, 0, 0, 0.5) inset;
                text-shadow: 0 1px 0 rgba(0, 0, 0, 0.5);
                border-radius: 2px;
                padding: 0.5vmin 0.8vmin;
                ",
      },
      "extend": "
            box-shadow: 0 1px 0 #495b71 inset;
            border-bottom: 1px solid #253246;
            border-radius: 2px 2px 0 0;
            ",
      "height": "5vmin",
      "icon": Object {
        "color": "#e0c671",
        "fontSize": "2.2vmin",
      },
      "subject": Object {},
    },
    "wrapper": Object {
      "padding": "0.6vmin",
    },
  },
}
`;
